### DISCUSSION TITLE
v1.3.6

### DISCUSSION THREAD

**Discussion by LazyImmortal** [LazyImmortal/Sesame#2] [Announcements]
## 特别鸣谢

- 该项目复刻自[Sesame-TK](https://github.com/TKaxv-7S/Sesame-TK)，感谢作者[@TKaxv-7S](https://github.com/TKaxv-7S)对代码进行重构
- 新界面开发自[@wh-990624](https://github.com/wh-990624)，感谢作者的前端设计，由于某闭源项目违反项目开源协议，前端作者已将新UI源码闭源

## 更新说明

- 修复 使用加速卡后，小鸡处于饥饿状态时，不立即进行投喂且蹲点投喂时间设置错误的问题

### Compatible 为兼容版本，适用于**Android 7.0**及以下的系统，最低支持**Android 5.1**

<hr /><em>This discussion was created from the release <a href='https://github.com/LazyImmortal/Sesame/releases/tag/v1.3.6'>v1.3.6</a>.</em>

**Comment by pidanpro** [comment]
![Screenshot_2024-07-24-23-34-04-847_io github lazyimmortal sesame](https://github.com/user-attachments/assets/0b5818cb-c4e7-4bfa-8371-0397e437b9d2)
仓库定向未更新。

**Comment by kingsley2021** [comment]
欢迎新大佬！请问能增加天天冲鸭的活动支持吗？谢谢
